gradle.properties

```diff
@@ -19,27 +19,27 @@
 
 pluginGroup = com.netflix.graphql.dgs
 pluginName = DGS
-pluginVersion = 1.4.2
+pluginVersion = 1.4.3
 
 # See https://plugins.jetbrains.com/docs/intellij/build-number-ranges.html
 # for insight into build numbers and IntelliJ Platform versions.
-pluginSinceBuild=242.1
+pluginSinceBuild=243
 
 # Plugin Verifier integration -> https://github.com/JetBrains/gradle-intellij-plugin#plugin-verifier-dsl
 # See https://jb.gg/intellij-platform-builds-list for available build versions.
-pluginVerifierIdeVersions = 2024.2.1
+pluginVerifierIdeVersions = 2024.3
 
 platformType = IU
-platformVersion = 2024.2.1
+platformVersion = 2024.3
 platformDownloadSources = true
 
 # Plugin Dependencies -> https://plugins.jetbrains.com/docs/intellij/plugin-dependencies.html
 # Example: platformPlugins = com.intellij.java, com.jetbrains.php:203.4449.22
-platformPlugins = com.intellij.lang.jsgraphql:242.20224.155
+platformPlugins = com.intellij.lang.jsgraphql:243.21565.122
 platformBundledPlugins = com.intellij.java, org.jetbrains.kotlin, com.intellij.gradle
 
 # Java language level used to compile sources and to generate the files for - Java 17 is required since 2023.1
```

Comment:
nit: I guess this comment is obsolete now (or update to reflect the latest state)